hack/aks/README.md

```diff
@@ -21,21 +21,23 @@ SWIFT Infra
   net-up           Create required swift vnet/subnets
 
 AKS Clusters
-  byocni-up                     Alias to swift-byocni-up
-  cilium-up                     Alias to swift-cilium-up
-  up                            Alias to swift-up
-  overlay-byocni-up             Bring up a Overlay BYO CNI cluster
-  overlay-byocni-nokubeproxy-up Bring up a Overlay BYO CNI cluster without kube-proxy
-  overlay-cilium-up             Bring up a Overlay Cilium cluster
-  overlay-up                    Bring up a Overlay AzCNI cluster
-  swift-byocni-up               Bring up a SWIFT BYO CNI cluster
-  swift-byocni-nokubeproxy-up   Bring up a SWIFT BYO CNI cluster without kube-proxy
-  swift-cilium-up               Bring up a SWIFT Cilium cluster
-  swift-up                      Bring up a SWIFT AzCNI cluster
-  windows-cniv1-up              Bring up a Windows AzCNIv1 cluster
-  linux-cniv1-up                Bring up a Linux AzCNIv1 cluster
-  dualstack-overlay-byocni-up   Bring up an dualstack overlay cluster without CNS and CNI installed
-  windows-nodepool-up           Add windows node pool
-  down                          Delete the cluster
-  vmss-restart                  Restart the nodes of the cluster
+  byocni-up                       Alias to swift-byocni-up
+  cilium-up                       Alias to swift-cilium-up
+  up                              Alias to swift-up
+  overlay-byocni-up               Bring up a Overlay BYO CNI cluster
+  overlay-byocni-nokubeproxy-up   Bring up a Overlay BYO CNI cluster without kube-proxy
+  overlay-cilium-up               Bring up a Overlay Cilium cluster
+  overlay-up                      Bring up a Overlay AzCNI cluster
+  swift-byocni-up                 Bring up a SWIFT BYO CNI cluster
+  swift-byocni-nokubeproxy-up     Bring up a SWIFT BYO CNI cluster without kube-proxy
+  swift-cilium-up                 Bring up a SWIFT Cilium cluster
+  swift-up                        Bring up a SWIFT AzCNI cluster
+  windows-cniv1-up                Bring up a Windows AzCNIv1 cluster
+  linux-cniv1-up                  Bring up a Linux AzCNIv1 cluster
+  dualstack-overlay-byocni-up     Bring up an dualstack overlay cluster without CNS and CNI installed
+  cilium-dualstack-up             Brings up a Cilium Dualstack Overlay cluster with Linux node only
+  dualstack-byocni-nokubeproxy-up Brings up a Dualstack overlay BYOCNI cluster with Linux node only and no kube-proxy
```

Comment:
nit: Capitalization on `overlay` -> `Overlay` 

nit: Should we use BYOCNI or BYO CNI?